javascript/statements.json

```diff
@@ -1008,11 +1008,11 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "13",
+              "version_added": "14",
               "notes": "Prior to Firefox 51, using the <code>for...of</code> loop construct with the <code>const</code> keyword threw a <code>SyntaxError</code> (\"missing = in const declaration\")."
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "14"
```

Comment:
Same as above + the note is likely to be copied too.

Reply:
I copied the note.